Libraries/WebSocket/RCTReconnectingWebSocket.m

```diff
@@ -19,6 +19,27 @@
 
 #if RCT_DEV // Only supported in dev mode
 
+// From https://github.com/apple/swift/blob/ad40c770bfe372f879b530443a3d94761fe258a6/stdlib/public/SDK/os/os_log.m
```

Comment:
It might be necessary to add a `__IPHONE_OS_VERSION_MAX_ALLOWED >= 100300` check here since the entire API around this (e.g. `_os_log_pack_size` etc.) has only been added in iOS 10.3, but React supports up to iOS 8.

Reply:
Added